### ISSUE TITLE
Orangutang SVG (svg/emoji_u1f9a7.svg) not rendering correctly 

### ISSUE THREAD

**Issue by jdherg** [googlefonts/noto-emoji#363]
The [orangutang SVG (svg/emoji_u1f9a7.svg)](https://github.com/googlefonts/noto-emoji/blob/main/svg/emoji_u1f9a7.svg) is broken, in what looks to be the same way it was broken before being fixed by 24a33c4 -- there's a reference to a jpg that doesn't actually exist.

**Comment by anthrotype** [comment]
Yes, I can confirm the orangutan has a badly broken arm, it looks like this when I load the [emoji_u1f9a7.svg](https://github.com/googlefonts/noto-emoji/blob/main/svg/emoji_u1f9a7.svg) in Chrome

![image](https://user-images.githubusercontent.com/6939968/134367240-094e39f0-81bb-4d3f-9c13-9fa13ded2bdc.png)

Inspecting it in a text editor reveals that it suddenly grew an unsupported `<image>` element that references an external (missing) JPEG file:


```
<image style="overflow:visible;" width="139" height="153" xlink:href="72D8D7011A0AC19D.jpg"  transform="matrix(0.48 0 0 0.48 59.52 -1.44)"></image>
```

Removing the line containing the `<image>` element still leaves the arm broken like so:

![image](https://user-images.githubusercontent.com/6939968/134368247-e65385e5-f98b-44b9-9624-55dc5725e7a1.png)

According to git blame the commit that first introduced this was https://github.com/googlefonts/noto-emoji/commit/9bdaf2e265ad7e8e81e65965a50ea91af6647be3 from 11 May this year (5 montsh ago)

**Comment by anthrotype** [comment]
A previous commit https://github.com/googlefonts/noto-emoji/commit/24a33c49 on `main` from November 2020 was meant to fix the orangutan (and two others) that had been broken in a similar way by an even older commit https://github.com/googlefonts/noto-emoji/commit/31f714f3. 
It looks like https://github.com/googlefonts/noto-emoji/commit/9bdaf2e265ad7e8e81e65965a50ea91af6647be3 involutarily reverted the orangutan to a broken state (with the arm pointing to a missing JPG image).

I suggest we revert it again to the good state from https://github.com/googlefonts/noto-emoji/commit/24a33c4 --  I'll send a PR
